src/transformers/models/roformer/tokenization_roformer.py

```diff
@@ -143,13 +158,13 @@ def __init__(
             )
         self.wordpiece_tokenizer = WordpieceTokenizer(vocab=self.vocab, unk_token=self.unk_token)
         try:
-            import rjieba
+            import jieba
         except ImportError:
             raise ImportError(
-                "You need to install rjieba to use RoFormerTokenizer."
-                "See https://pypi.org/project/rjieba/ for installation."
+                "You need to install jieba to use RoFormerTokenizer."
+                "See https://pypi.org/project/jieba/ for installation."
             )
-        self.jieba = rjieba
+        self.jieba = jieba
```

Comment:
Is this the correct way of handling the rjieba dependency @LysandreJik?

Reply:
We decided to handle it this way as it's the only model that requires it - and if other models arrive, then to upstream it like it is done for the other models.